### PR TITLE
Weather stencil with AIR

### PR DESCRIPTION
**Pull request by singagan**

### CODE REVIEW

File: examples/weather_stencil_air/aie_4.mlir [hunk 1]
```diff
@@ -0,0 +1,1561 @@
+// (c) 2023 SAFARI Research Group at ETH Zurich, Gagandeep Singh, D-ITET   
+
+// SPDX-License-Identifier: MIT
+
+
+module @hdiff_bundle_4 {
```
Comment (eddierichter-amd) [inline examples/weather_stencil_air/aie_4.mlir:6]: We should make this one offset at column 2 as well.

File: examples/weather_stencil_air/four_bblock_output.txt
Comment (eddierichter-amd) [inline examples/weather_stencil_air/four_bblock_output.txt:1]: We can probably delete this, or at least update it with the new values.

File: examples/weather_stencil_air/hdiff_flux1.cc [hunk 2]
```diff
@@ -0,0 +1,115 @@
+// (c) 2023 SAFARI Research Group at ETH Zurich, Gagandeep Singh, D-ITET
+
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
```
Comment (eddierichter-amd) [inline examples/weather_stencil_air/hdiff_flux1.cc:3]: Think this snuck through. Guessing it is also the case for the other kernels.

File: examples/weather_stencil_air/hdiff_flux2.cc [hunk 3]
```diff
@@ -0,0 +1,104 @@
+// (c) 2023 SAFARI Research Group at ETH Zurich, Gagandeep Singh, D-ITET
+
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
```
Comment (eddierichter-amd) [inline examples/weather_stencil_air/hdiff_flux2.cc:3]: Note.

File: examples/weather_stencil_air/single_bblock_output.txt
Comment (eddierichter-amd) [inline examples/weather_stencil_air/single_bblock_output.txt:1]: Again, we can probably remove this now.

**Review by eddierichter-amd — CHANGES_REQUESTED** [review]
Couple small updates.

**Comment by singagan** [comment]
Fixed column offset and headers. Also, removed old output files.

**Review by eddierichter-amd — APPROVED** [review]
LGTM

**Pull request merged**